### DISCUSSION TITLE
验证推理模型时只识别检测框中的一部分字符

### DISCUSSION THREAD

**Discussion by 555myhair** [PaddlePaddle/PaddleOCR#13493] [Q&A]
Hi，各位
    我分别测试检测模型和识别模型时，识别率很高，转换成推理模型后，检测结果显示一大半的照片只识别了检测框的一部分内容。检测框是把内容都框在内的。
![屏幕截图 2024-07-25 093929](https://github.com/user-attachments/assets/97812e5d-0343-470b-8bb3-e6b51888e7b9)

**Comment by GreatV** [comment]
你的识别模型是不是不是动态shape的

**Reply by 555myhair** [reply]
> 你的识别模型是不是不是动态shape的

似乎是的，我的配置文件是这样的：image_shape: [3, 48, 320]
那我应该怎么改呢？

**Reply by UserWangZz** [reply]
推理时，有一个保存crop图片的参数，尝试看看送入识别模型的图片是否正确，看看对文本区域的crop正常不

**Comment by 555myhair** [comment]
我训练生成的模型是rec。我怎么样导出的是动态图的模型？
